src/main/java/HTTP/HTTPClient.java

```diff
@@ -0,0 +1,32 @@
+package HTTP;
+
+import JSON.JSONHelper;
+import lombok.SneakyThrows;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+
+public class HTTPClient {
+
+    @SneakyThrows
+    public HTTPResponseDTO callPostReturnHTTPResponseData(CloseableHttpClient httpClient, String URL) {
+
+        HTTPResponseDTO HTTPResponseDTO = new HTTPResponseDTO();
+        JSONHelper jsonHelper = new JSONHelper();
+
+        HttpPost httpPost = new HttpPost(URL);
+        CloseableHttpResponse httpResponse = httpClient.execute(httpPost);
+        HTTPResponseDTO.setStatus(String.valueOf(httpResponse.getStatusLine().getStatusCode()));
+        try {
+            HttpEntity entity = httpResponse.getEntity();
+            if (entity != null) {
+                HTTPResponseDTO.setBody(jsonHelper.convertResponseBodyIntoJSON(entity));
```

Comment:
It will be good if you use objectMapper class from the Jackson library. It will save you time in the future. https://www.baeldung.com/jackson-object-mapper-tutorial